internal/controller/controllers/infraenv_controller.go

```diff
@@ -323,6 +324,11 @@ func (r *InfraEnvReconciler) ensureISO(ctx context.Context, log logrus.FieldLogg
 	var inventoryErr, err error
 	var Requeue bool
 	var cluster *common.Cluster
+	// Retrieve infraenv from the database
```

Comment:
Why move this? Now the comment makes no sense. Either remove the comment (it probably didn't need to be there in the first place) or put this variable back where it was.